examples/flutter_gallery/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ dependencies:
 
   flutter:
     sdk: flutter
+  url_launcher: ^0.3.1
```

Comment:
You could also just do `<1.0.0`.

But generally we're planning on moving to pinning the dependencies exactly, so I'd just say `0.3.1`.